lms/templates/dashboard.html

```diff
@@ -206,7 +206,7 @@
                 cert_status = cert_statuses.get(session_id)
                 can_refund_entitlement = entitlement and entitlement.is_entitlement_refundable()
                 partner_managed_enrollment = enrollment.mode == 'masters'
-                can_unenroll = False if partner_managed_enrollment else (not cert_status) or cert_status.get('can_unenroll') if not unfulfilled_entitlement else False
+                can_unenroll = (False if partner_managed_enrollment else (not cert_status)) or (cert_status.get('can_unenroll') if not unfulfilled_entitlement else False) and (not disable_unenrollment)
```

Comment:
@tinumide, imho, this statement is very long and has multiple conditions combined by different operators. So it is difficult to read and understand easily. Can you break it down or simplify it?

@tinumide